vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt

```diff
@@ -1031,26 +1031,28 @@ class RoomDetailFragment @Inject constructor(
 
     private fun updateJumpToReadMarkerViewVisibility() {
         views.jumpToReadMarkerView.post {
-            withState(roomDetailViewModel) {
-                val showJumpToUnreadBanner = when (it.unreadState) {
-                    UnreadState.Unknown,
-                    UnreadState.HasNoUnread            -> false
-                    is UnreadState.ReadMarkerNotLoaded -> true
-                    is UnreadState.HasUnread           -> {
-                        if (it.canShowJumpToReadMarker) {
-                            val lastVisibleItem = layoutManager.findLastVisibleItemPosition()
-                            val positionOfReadMarker = timelineEventController.getPositionOfReadMarker()
-                            if (positionOfReadMarker == null) {
-                                false
+            if (isAdded) {
```

Comment:
I would have replaced the post and the isAdded blocks by
` viewLifecycleOwner.lifecycleScope.launchWhenResumed`

Reply:
Yes, better, thanks! Done